packages/lesswrong/server/vulcan-lib/apollo-server/initGraphQL.ts

```diff
@@ -543,6 +550,7 @@ export const resolvers = {
     ...userTagRelGqlQueryHandlers,
     ...userGqlQueryHandlers,
     ...voteGqlQueryHandlers,
+    ...bookmarkGqlQueryHandlers,
```

Comment:
generalized feedback: not every stylistic convention can realistically be enforced via code (i.e. lint rule).  nonetheless it can be good to keep things in alphabetical order.